src/devices_models/devices/thermal_generation.jl

```diff
@@ -28,6 +28,7 @@ get_variable_initial_value(::ActivePowerVariable, d::PSY.ThermalGen, ::AbstractT
 get_variable_initial_value(::PowerAboveMinimumVariable, d::PSY.ThermalGen, ::AbstractCompactUnitCommitment) = max(0.0, PSY.get_active_power(d) - PSY.get_active_power_limits(d).min)
 
 get_variable_lower_bound(::ActivePowerVariable, d::PSY.ThermalGen, ::AbstractThermalFormulation) = PSY.get_active_power_limits(d).min
+get_variable_lower_bound(::ActivePowerVariable, d::PSY.ThermalGen, ::AbstractThermalUnitCommitment) = 0.0
```

Comment:
So in the previous implementation of range constraints, we would grab the variable and remove its bounds if we added a binary variable based lower bound. Now with the possibility of having either variables or expressions, its better to set the bound to 0.0 here and changing it later

Reply:
ok